### PR TITLE
HOCS-2193-Private Office Team target/deadline (Mark up)

### PR DESCRIPTION
**Pull request by agraham19**
* add steps to update deadlines for ministers/directors in markup
* add unit tests for DCU_MIN_MARKUP

### CODE REVIEW

File: src/main/resources/processes/DCU_MIN_MARKUP.bpmn [hunk 1]
```diff
@@ -175,11 +175,6 @@
       <bpmn:incoming>SequenceFlow_1r3bhri</bpmn:incoming>
       <bpmn:outgoing>SequenceFlow_14u7a2d</bpmn:outgoing>
     </bpmn:serviceTask>
-    <bpmn:serviceTask id="ServiceTask_1wwfdzr" name="Update Teams for Topic" camunda:expression="${bpmnService.updateTeamsForPrimaryTopic(execution.getVariable(&#34;CaseUUID&#34;),execution.processBusinessKey, execution.getVariable(&#34;Topics&#34;), &#34;DCU_MIN_INITIAL_DRAFT&#34;, &#34;DraftingTeamUUID&#34;, &#34;DraftingTeamName&#34; )}">
```
Comment (MarkWrightBjss) [inline src/main/resources/processes/DCU_MIN_MARKUP.bpmn:178]: I'm not sure why this has been removed.

**Comment by MarkWrightBjss** [comment]
Story says "As a user at Mark Up and Private office sign-off when I change the Private Office Team..."
I can see that this PR changes Mark Up, but what about Private Office?

**Comment by agraham19** [comment]
> Story says "As a user at Mark Up and Private office sign-off when I change the Private Office Team..."
> I can see that this PR changes Mark Up, but what about Private Office?

Ah. I'll add that sorry.

File: src/main/resources/processes/DCU_MIN_MARKUP.bpmn (hunk 1 above)
Comment (agraham19) [reply src/main/resources/processes/DCU_MIN_MARKUP.bpmn:178]: I'm not sure. I never intentionally deleted it, but I have replaced it.

**Comment by agraham19** [comment]
> > Story says "As a user at Mark Up and Private office sign-off when I change the Private Office Team..."
> > I can see that this PR changes Mark Up, but what about Private Office?
> 
> Ah. I'll add that sorry.

I'll write the tests for the private office flow before I make changes, so this doesn't happen again.

**Comment by agraham19** [comment]
> Story says "As a user at Mark Up and Private office sign-off when I change the Private Office Team..."
> I can see that this PR changes Mark Up, but what about Private Office?

I have added this

**Comment by philbarton** [comment]
Looks ok to me. Test Coverage is good.

File: src/test/java/uk/gov/digital/ho/hocs/workflow/processes/DCU_MIN_PRIVATE_OFFICE.java [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+                eq("DCU_MIN_NO_REPLY_NEEDED_CONFIRMATION"), eq("10"),
+                eq("DCU_MIN_DISPATCH"), eq("10"),
+                eq("DCU_MIN_COPY_NUMBER_TEN"), eq("10")
+        );
+    }
+
+    private void verifyMinisterOrDirectorOfficeDeadlinesSet() {
+        verify(dcuMinPrivateOffice).hasCompleted(UPDATE_MINISTER_OR_DIRECTOR_DEADLINE);
+
+        // Case deadline of 10 days
```
Comment (MarkWrightBjss) [inline src/test/java/uk/gov/digital/ho/hocs/workflow/processes/DCU_MIN_PRIVATE_OFFICE.java:229]: Fix or remove this comment.

**Review by MarkWrightBjss — CHANGES_REQUESTED** [review]

File: src/test/java/uk/gov/digital/ho/hocs/workflow/processes/DCU_MIN_PRIVATE_OFFICE.java (hunk 2 above)
Comment (agraham19) [reply src/test/java/uk/gov/digital/ho/hocs/workflow/processes/DCU_MIN_PRIVATE_OFFICE.java]: I have addressed this.

Comment (MarkWrightBjss) [reply src/test/java/uk/gov/digital/ho/hocs/workflow/processes/DCU_MIN_PRIVATE_OFFICE.java]: Good stuff, thanks.

**Review by MarkWrightBjss — APPROVED** [review]

**Pull request merged**